persist-cli/src/main/java/io/ballerina/persist/cmd/Generate.java

```diff
@@ -177,6 +184,8 @@ public void execute() {
                 sourceCreator.createGSheetSources();
                 errStream.printf("Generated Ballerina Client, and Types to %s directory.%n", generatedSourceDirPath);
                 errStream.println("You can now start using Ballerina Client in your code.");
+                errStream.printf("If your spreadsheet has no sheets yet, execute the scripts." +
```

Comment:
this should come under Next step

Reply:
Next steps:
Execute the "scripts.js" file located at %s directory in your worksheet to create sheets.```